src/content/docs/kubernetes-pixie/k8s-otel/advance-config.mdx

```diff
@@ -0,0 +1,282 @@
+---
+title: Advanced configuration
+metaDescription: Learn how to configure charts for advanced use cases in Kubernetes with OpenTelemetry.
+tags:
+    - Kubernetes
+    - OpenTelemetry
+    - Advanced Configuration
+    - New Relic Kubernetes integration
+    - New Relic Distribution on OpenTelemetry
+    - NRDOT
+freshnessValidatedDate: never
+---
+
+New Relic provides [Helm chart](https://github.com/newrelic/helm-charts/blob/master/charts/nr-k8s-otel-collector/README.md) to deploy OpenTelemetry Collector in your Kubernetes cluster. This Helm charts can be customized to suit your specific needs, including advanced configurations for various use cases.
+
+There are several advanced configuration options available for the `nr-k8s-otel-collector` Helm chart. These options allow you to customize the behavior of the OpenTelemetry Collector to better suit your needs. The chart supports both `DaemonSet` and `Deployment` collectors, allowing you to choose the best fit for your use case. For more information on installing the New Relic OpenTelemetry Collector in Kubernetes, refer to the [installation guide](/docs/kubernetes-pixie/k8s-otel/install).
+
+This document provides an overview of some of the key advanced configuration options you can use.
+
+
+## Enable GKE Autopilot or Red Hat OpenShift Compatibility [#enable-provider]
+
+
+To ensure compatibility with specific Kubernetes environments, you can enable provider-specific configurations. This setting ensures compatibility and proper functionality of the OpenTelemetry Collectors by adapting to the specific constraints of these environments. You can enable this option in your [`values.yaml`](/docs/kubernetes-pixie/k8s-otel/install/#config-params) file:
+
+```yaml
+  provider: "GKE_AUTOPILOT" # Or "OPEN_SHIFT" if applicable
+```
+
+
+## Enable Low Data Mode [#low-data]
+
+
+The `LowDataMode` option is enabled by default to ingest only the metrics required by our Kubernetes UIs. This mode reduces the amount of data collected, focusing on essential metrics for Kubernetes monitoring. To fetch additional metrics, you can add new pipelines and configuring the appropriate receivers and processors in your [`values.yaml`](/docs/kubernetes-pixie/k8s-otel/install/#config-params) file using the `extraConfig` section.
+
```

Comment:
I'm missing a subtitle like "How to add additional metrics together with LowDataMode" so customers can easily identify the content